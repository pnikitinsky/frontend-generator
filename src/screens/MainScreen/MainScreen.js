import React from 'react';
import styled from 'styled-components';

const Container = styled.div`
  display: flex;
  align-items: center;
  justify-content: space-between;
  flex-direction: column;
  overflow: auto;
  max-height: 100vh;
`;


export default class MainScreen extends React.Component {
  static propTypes = {
  };


  render() {
    return (
      <Container/>
    );
  }
}
