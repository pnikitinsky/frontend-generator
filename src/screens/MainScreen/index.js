export { default as Component } from './MainScreen';
export { default as Container } from './MainScreenContainer';