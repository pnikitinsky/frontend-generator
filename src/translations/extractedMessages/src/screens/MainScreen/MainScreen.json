[
  {
    "id": "main.screen.hello",
    "defaultMessage": "Hello World!"
  }
]