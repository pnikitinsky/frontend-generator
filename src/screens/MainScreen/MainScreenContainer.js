import { connect } from 'react-redux';
import { bindActionCreators } from 'redux';
import MainScreen from './MainScreen';

export function mapStateToProps(state) {
  return {

  };
}

export function mapDispatchToProps(dispatch) {
  return bindActionCreators({
  }, dispatch);
}

export default connect(
  mapStateToProps,
  mapDispatchToProps
)(MainScreen);
