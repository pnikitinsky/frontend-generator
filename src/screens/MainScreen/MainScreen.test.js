import React from 'react';
import MainScreen from './MainScreen';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';

test('MainScreen should match snapshot', () => {
  const component = renderer.create(
    <MemoryRouter>
      <MainScreen/>
    </MemoryRouter>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
