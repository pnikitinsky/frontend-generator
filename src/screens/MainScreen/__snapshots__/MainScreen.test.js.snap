// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainScreen should match snapshot 1`] = `
<div
  className="sc-bdVaJa grbsvW"
>
  <span>
    <span>
      Hello World!
    </span>
  </span>
  <a
    href="/test"
    onClick={[Function]}
  >
    Take me to another screen!
  </a>
</div>
`;
